src/common.scss

```diff
@@ -0,0 +1,22 @@
+/* stylelint-disable */
```

Comment:
- [ ] I noticed that you disable a lot **Linters* . Please remove the **Linters disable** statement and try to fix all the errors you are getting. Because ` Linters` helps developers to write clean codes and adapt best practices when writing the code.  :+1: